### ISSUE TITLE
Getting module not found error in project_creation

### ISSUE THREAD

**Issue by the-indian-saint** [GoogleCloudPlatform/deploymentmanager-samples#223]
When I try to use the project creation template which is on github, even after changing the values in config.yaml I am getting following error.

location: /deployments/projectcreation000/manifests/manifest-1534790908361 message: 'Manifest expansion encountered the following errors: Error compiling Python code: No module named apis Resource: project.py Resource: config'

you can find the repo link here : https://github.com/GoogleCloudPlatform/deploymentmanager-samples/tree/master/examples/v2/project_creation

Please help as I need it for production workflow. I have tried "sudo pip install apis" in Cloud Shell but it does not help, even after successful installation of apis module.

**Comment by thackem** [comment]
The apis module in this context refers to:
https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/examples/v2/project_creation/apis.py

Not a pip package. Ensure you have all the files in the same relative paths to each other when deploying these samples.

**Comment by the-indian-saint** [comment]
Thank for the reply. But now I am getting below error.

 message: '{"ResourceType":"cloudresourcemanager.v1.project","ResourceErrorCode":"403","ResourceErrorMessage":{"code":403,"message":"User
    is not authorized.","status":"PERMISSION_DENIED","statusMessage":"Forbidden","requestPath":"https://cloudresourcemanager.googleapis.com/v1/projects","httpMethod":"POST"}}'

I have assigned all the IAM roles to the service account of the DM-Creation-Project. and I am using the root account  in the CLI.

**Comment by ocsig** [comment]
Have you gave the 'roles/resourcemanager.projectCreator' to the service account on the organization node?
Also are you sure that your project name is globally (not just in your org) unique?

**Comment by the-indian-saint** [comment]
Yes, I have assigned the role. I even tried making the service account the
administrator of the Organisation. but the issue persists.

**Comment by thackem** [comment]
If you go to
https://pantheon.corp.google.com/iam-admin/iam?organizationId=$PARENT_ORG_ID

Then filter by your DM service account:
$DM_PROJECT_NUMBER@cloudservices.gserviceaccount.com

Ensure you have the "Project Creator" role assigned

**Comment by SimyungYang** [comment]
For the above problem, I couldn't find anywhere import apis.py in project_creation project.
After I added the below line, the error message is gone.

==============
imports:
- path: project.py
- path: apis.py
==============

I'm stuck with another error message.

message: '{"ResourceType":"cloudresourcemanager.v1.project","ResourceErrorCode":"403","ResourceErrorMessage":{"code":403,"message":"User
    is not authorized.","status":"PERMISSION_DENIED","statusMessage":"Forbidden","requestPath":"https://cloudresourcemanager.googleapis.com/v1/projects","httpMethod":"POST"}}'


https://console.cloud.google.com/iam-admin/iam?organizationId=$PARENT_ORG_ID

If I visit the url above, only account I can see is my team members' account, not any of service accounts. 
I don't know how to give projectCreator role to DM service account.
SO I give "Organization Administrator" role to it.
Can anyone share the howto set project Creator to service account?

Although I give "Organization Administrator" role to the account with gcloud cli and DM service account, I've got Unauthorized error.
